### PR TITLE
[travis] run on all pull requests

### PR DESCRIPTION
**Pull request by Saviq**
This way we can chain PRs behind one another.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,11 +6,9 @@ language: cpp
 git:
   depth: false
 
-branches:
-  only:
-  - staging
-  - trying
-  - master
+if: |
+  branch IN (staging, trying, master)
```
Comment (ricab) [inline .travis.yml]: doesn't this need a slash to continue the line? https://docs.travis-ci.com/user/conditions-v1#line-continuation-multiline-conditions

Comment (ricab) [reply .travis.yml]: according to [this](https://yaml-multiline.info/) newlines would be kept in the string with `|`

Comment (ricab) [reply .travis.yml]: However, Travis seems happy...

Comment (Saviq) [reply .travis.yml]: Yeah I think the important bit is "We liked the idea, so we allowed the following:"... FWIW [these](https://github.com/MirServer/mir/blob/master/.travis.yml#L15) have been working for a while.

**Comment by ricab** [comment]
bors r+ 

I know this will fail for now for unrelated reasons, but it might detect other problems here.

**Comment by ricab** [comment]
bors delegate=Saviq

**Pull request merged**